src/utils/get-image-from-ans/index.js

```diff
@@ -1,12 +1,13 @@
+// returns an object
 const extractImageFromPromo = (promoItem) =>
-	promoItem?.basic?.type === "image" && promoItem?.basic?.url ? promoItem.basic.url : null;
+	promoItem?.basic?.type === "image" && promoItem?.basic?.url ? promoItem.basic : null;
```

Comment:
```suggestion
	promoItem?.basic?.type === "image" ? promoItem.basic : null;
```

Reply:
updated -- makes sense

Reply:
will update the test expectations if we're changing the validation here

Reply:
updated

Reply:
addressed